### ISSUE TITLE
Create new Amount component

### ISSUE THREAD

**Issue by tiagoalvesdulce** [decred/pi-ui#386]
<img width="633" alt="Screen Shot 2021-11-16 at 10 56 41" src="https://user-images.githubusercontent.com/13955303/141998502-d4fbbb5c-2a0d-4279-89a0-f4964fd48900.png">

This new component can be a variation of NumberInput.

@MariaPleshkova @linnutee Question: the button in the old Amount is responsible for filling the Amount with all the funds in the wallet. I think this is pretty useful for moving funds from default to unmixed, for example. How would something like that be accomplished with the new design?

**Comment by linnutee** [comment]
Yep I think so, amount picker is important enough to have a distinct field.

**Comment by vibros68** [comment]
Can I see the origin design @tiagoalvesdulce ?

**Comment by tiagoalvesdulce** [comment]
I took this component from the figma master file here: https://github.com/decred/pi-ui/issues/323. This is more of a decrediton component so I think decrediton devs should handle it. I'll create labels to make this clear.

**Comment by amass01** [comment]
@tiagoalvesdulce @alexlyp @bgptr I would like to work on this one if it still in todo ?

**Comment by tiagoalvesdulce** [comment]
If the decrediton devs think this component will still be useful, go for it!

**Comment by bgptr** [comment]
Sure, go for it!

**Comment by amass01** [comment]
hrmmm looks like storybook doesn't work 💯 ATM, how do you test your pi-ui work guys ?

**Comment by tiagoalvesdulce** [comment]
@amass01 sorry for the delay. I didn't get a notification somehow. Can you detail your issue? Storybook is working here just fine.
